server/budgettracker.business/Api/BudgetApi.cs

```diff
@@ -29,15 +30,27 @@ public BudgetApi(IBudgetRepository budgetRepository, IConfiguration appConfig, U
                     ConfigurationReader.FromAppConfiguration(appConfig))
         {
             _budgetRepository = budgetRepository;
+            _budgetConverter = new BudgetApiConverter();
         }
 
         public async Task<ApiResponse> CreateBudget(ApiRequest request)
         {
             Authenticate(request);           
             
-            await _budgetRepository.CreateBudget(_budgetConverter.ToModel(request.Arguments<CreateBudgetRequestContract>()));
-
-            CreateBudgetResponseContract response = _budgetConverter.ToResponseContract(_budgetConverter.ToModel(request.Arguments<CreateBudgetRequestContract>()));
+            Budget newBudget = _budgetConverter.ToModel(request.Arguments<CreateBudgetRequestContract>());
+
+            newBudget.Id = Guid.NewGuid();
+            newBudget.BudgetStart = new DateTime();
```

Comment:
Maybe for future reference we'll want to let the user decide when the budget starts but for now this is good.

Reply:
#43 and #44